packages/teleport/src/Databases/Databases.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { useState } from 'react';
+import {
+  FeatureBox,
+  FeatureHeader,
+  FeatureHeaderTitle,
+} from 'teleport/components/Layout';
+import { Flex } from 'design';
+import InputSearch from 'teleport/components/InputSearch';
+import DatabaseList from 'teleport/components/DatabaseList';
+
+export default function Databases(props) {
+  const { databases } = props;
+
+  const [searchValue, setSearchValue] = useState('');
+
+  return (
+    <FeatureBox>
+      <FeatureHeader alignItems="center" justifyContent="space-between">
+        <FeatureHeaderTitle>Databases</FeatureHeaderTitle>
+      </FeatureHeader>
+      <Flex
+        mb={4}
+        alignItems="center"
+        flex="0 0 auto"
+        justifyContent="space-between"
+      >
+        <InputSearch
+          mr="3"
+          onChange={e => {
+            setSearchValue(e);
+          }}
+        />
+      </Flex>
```

Comment:
Lets put this inside the list component b/c the search box is for the table (i looked up NodesList and realized that the search box wasn't a part of the component, it looks like it was extracted for flexibility of placement b/c in `DocumentNodes` we have two input boxes to show above the table: search and quick launcher). 

Lookup `UserList.tsx` or `RequestList.tsx` and you'll see how it is done there.